sklearn/metrics/tests/test_dist_metrics.py

```diff
@@ -89,6 +89,8 @@ def check_cdist(metric, kwargs, X1, X2):
     assert_array_almost_equal(D_sklearn, D_scipy_cdist)
 
 
+# TODO: Remove filterwarnings in 1.3 when wminkowski is removed
+@pytest.mark.filterwarnings("ignore::FutureWarning:sklearn")
```

Comment:
Could you please adjust all those `filterwarnings` conditions to only filter warnings that have a message that contains  "WMinkowskiDistance"?

```suggestion
@pytest.mark.filterwarnings("ignore:WMinkowskiDistance:FutureWarning:sklearn")
```